src/zone/timer.cpp

```diff
@@ -67,37 +67,37 @@ bool TimersScheduler::StartMainLoop() {
         // Timers main loop.
         while (running) {
             cv.wait(lock, [this]() {
-                return !activeTimers.empty() || !running;
+                return !this->activeTimers.empty() || !this->running;
```

Comment:
Is this fixing code style consistency?

Reply:
Yeah. OK to make it consist with old style.

Reply:
Keep it in old style.